src/pybind/mgr/cephadm/module.py

```diff
@@ -1519,22 +1519,15 @@ def list_daemons(self,
 
     @trivial_completion
     def service_action(self, action, service_name) -> List[str]:
-        args = []
-        for host, dm in self.cache.daemons.items():
-            for name, d in dm.items():
-                if d.matches_service(service_name):
-                    args.append((d.daemon_type, d.daemon_id,
-                                 d.hostname, action))
+        if action == 'stop':
```

Comment:
Why we cannot stop all the daemons of a service?. Is because we need to implement the ok_to_stop in each single service?

Reply:
* if you call `ceph orch stop osd`, you cluster is essentially down.
* if you call `ceph orch stop mon`, the cluster will hang indefinitely and the command will never complete
* if you call `ceph orch stop mgr`, TBH I actually don't know what will happen.
* if you call `ceph orch stop mds`, you cephfs will be down.

I can see that this might be useful for some use cases, but we need to make sure, users know what they are doing.